.github/workflows/release-next.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and Deploy
+on:
+  # Trigger the workflow on push only for the master branch
+  push:
+    branches:
+      - master
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
+        with:
+          persist-credentials: false
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10'
+      - name: Generate tag utilities
+        id: TAG_UTIL
+        run: |
+          echo "::set-output name=short_sha::$(git rev-parse --short HEAD)"
+          echo "::set-output name=current_hour::$(date +'%Y%m%d%H%M%S')"
+          echo "::set-output name=current_date::$(date +'%Y%m%d')"
+      - name: Install and Build 🔧 # This example project is built using npm and outputs the result to the 'build' folder. Replace with the commands required to build your project, or remove this step entirely if your site is pre-built.
+        run: |
+          git config --local user.name "Flavius Lacatusu"
```

Comment:
I think we should use there the author that merge to master

Reply:
Right!

Reply:
@benoitf How can I get the author in GH actions?

Reply:
{{ github.actor }} ?